packages/calcite-components/src/components/progress/progress.scss

```diff
@@ -6,8 +6,12 @@
 * These properties can be overridden using the component's tag as selector.
 *
 * @prop --calcite-progress-background-color: The background color of the component.
-* @prop --calcite-progress-font-size: The font size of the component.
 * @prop --calcite-progress-color: The font color of the component.
+* @prop --calcite-progress-font-size: The font size of the component.
+*
+* @prop —-calcite-progress-label-inline-padding: Component label inline padding
```

Comment:
Should these be `internal`? Doesn't seem like we'd want to expose these publicly.